cypress/integration/nav_bar_home_button_spec.js

```diff
@@ -0,0 +1,7 @@
+describe("Navbar home button", () => {
+  it("navigates to the home page", () => {
+    cy.visit("/users/new");
+    cy.get("#home").click();
+    cy.url().should("include", "/?");
```

Comment:
I think the test should be equal http://localhost:3030/ , simon knows the exact syntax